tests/test_csc.py

```diff
@@ -0,0 +1,58 @@
+import asynctest
+# import unittest
+from lsst.ts import salobj
+from lsst.ts import Dome
+
+
+class CscTestCase(salobj.BaseCscTestCase, asynctest.TestCase):
```

Comment:
Do you want to go ahead and make a `test_basic_state_transition` to your CSC?

Reply:
Good idea indeed. Is there an example that I can follow?